### DISCUSSION TITLE
Show a loader when the AI is generating content

### DISCUSSION THREAD

**Discussion by kavir10** [briansunter/logseq-plugin-gpt3-openai#188] [Ideas]
When using the Cmd+J shortcut or when using the gpt-block command, show a loader to indicate to the user that the AI is working.

**Comment by briansunter** [comment]
Hey @kavir10, would using the GPT popup with a continue command suit your use case? That should be the same as gpt-block

**Comment by kavir10** [comment]
I prefer the Cmd+J shortcut for gpt-block because it is quicker. But when I don’t receive any system feedback that the AI is running, then I wonder if I used the shortcut properly and may run it again. 

After using Obsidian's plugin for GPT3, I got used to the loader.

**Comment by taariy** [comment]
Dear @briansunter please add a loader icon at the cursor location for "ctrl+j" to improve the slow load time with gpt-4. This would provide very much needed real-time feedback on the load process and enhance the user experience hugly. Thanks for your work.
